leveldown.js

```diff
@@ -38,6 +40,8 @@ function Multilevel (path, opts) {
 
 util.inherits(Multilevel, abstract.AbstractLevelDOWN)
 
+Multilevel.prototype.type = 'multileveldown'
```

Comment:
This is for compatibility with [`reachdown`](https://github.com/vweevers/reachdown), which I also intend to start using in `subleveldown`, `level-test` and more.